packages/platform-browser/src/dom/dom_renderer.ts

```diff
@@ -275,8 +276,10 @@ class DefaultDomRenderer2 implements Renderer2 {
 const AT_CHARCODE = (() => '@'.charCodeAt(0))();
 function checkNoSyntheticProp(name: string, nameKind: string) {
   if (name.charCodeAt(0) === AT_CHARCODE) {
-    throw new Error(`Found the synthetic ${nameKind} ${
-        name}. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application.`);
+    throw new Error(`Found the synthetic ${nameKind} ${name}. Possible reasons:
+  - "BrowserAnimationsModule" or "NoopAnimationsModule" are missing in your application.
+  - There is no corresponding configuration for the animation named \`${
+        name}\` defined in the \`animations\` field of the \`@Component\` decorator (see https://angular.io/api/core/Component#animations).`);
```

Comment:
I'd propose rephrasing it a bit:

```suggestion
    throw new Error(`Unexpected synthetic ${nameKind} ${name} found. Please make sure that:
  - Either "BrowserAnimationsModule" or "NoopAnimationsModule" are imported in your application.
  - There is corresponding configuration for the animation named \`${
        name}\` defined in the \`animations\` field of the \`@Component\` decorator (see https://angular.io/api/core/Component#animations).`);
```